cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -253,62 +276,20 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 	}
 	metrics.UpdateUnschedulablePodsCount(len(allUnschedulablePods))
 
-	allScheduled, err := scheduledPodLister.List()
+	allScheduledPods, err := scheduledPodLister.List()
 	if err != nil {
 		klog.Errorf("Failed to list scheduled pods: %v", err)
 		return errors.ToAutoscalerError(errors.ApiCallError, err)
 	}
 
-	allUnschedulablePods, allScheduled, err = a.processors.PodListProcessor.Process(a.AutoscalingContext, allUnschedulablePods, allScheduled, allNodes)
-	if err != nil {
-		klog.Errorf("Failed to process pod list: %v", err)
-		return errors.ToAutoscalerError(errors.InternalError, err)
-	}
+	ConfigurePredicateCheckerForLoop(allUnschedulablePods, allScheduledPods, a.PredicateChecker)
```

Comment:
Possibly we can reconfigure it after PLP.
It needs to be configured based on original pods before PLPs because PLPs can (and do) use predicate checker

Reply:
You're right. Let's leave it the way it is in this PR and we can consider adding additional call in a follow up PR if we find it's needed.